aten/src/ATen/native/BlasUtils.h

```diff
@@ -0,0 +1,45 @@
+#include <TH/THBlas.h>
+#include <ATen/ScalarType.h>
+
+// This header file shouldn't be anything permanent; it's just a temporary
```

Comment:
Could just move this specialization into non-generic TH (hpp). I did something similar with the Copy functions in THC, that way you don't have to invent new ATen APIs.  But this looks fine to me if you want to keep it.

Reply:
Moving into THC sounds fine.